examples/adolc_example.jl

```diff
@@ -0,0 +1,72 @@
+using FrankWolfe
+using ADOLC
+using LinearAlgebra
+using Test
+
+## Data setup
+n = Int(10000)
+
+xpi = rand(n);
+total = sum(xpi);
+const xp = xpi ./ total;
+
+f(x) = sum(abs2, x-xp)
+
+println("Automatic differentiation")
+
+const tape_id = 1
+ADOLC.derivative(f, zeros(n), :jac, tape_id=tape_id)
```

Comment:
maybe an explanation on the line here, since this call is not returning any variable